composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": ">=5.4.0",
         "illuminate/support": ">=5.0.0",
-        "onelogin/php-saml": "^2.10"
+        "onelogin/php-saml": "^3.0.0"
```

Comment:
Is there a composer.lock change?

Reply:
there is no composer.lock file

Reply:
It is ignored in .gitignore...I could commit it I suppose

Reply:
It says the lock file wouldn't really have an effect for libraries so that's probably why it's not included
https://getcomposer.org/doc/02-libraries.md#lock-file

Reply:
:+1: